src/server/bitops_family.cc

```diff
@@ -183,60 +287,203 @@ bool SetBitValue(uint32_t offset, bool bit_value, std::string* entry) {
 // ------------------------------------------------------------------------- //
 // Helper functions to access the data or change it
 
-class OverrideValue {
+class ElementAccess {
+  bool added_ = false;
+  PrimeIterator element_iter_;
+  std::string_view key_;
+  DbIndex index_ = 0;
+  EngineShard* shard_ = nullptr;
+
+ public:
+  ElementAccess(std::string_view key, DbIndex index) : key_{key}, index_{index} {
+  }
+
+  OpStatus Find(EngineShard* shard) {
+    try {
```

Comment:
done

Reply:
Also added a test for other function to ensure that if this failed, you cannot try and assign with it, so that you're getting an assertion if find was not called or failed.